scripts/run_e2e_tests.py

```diff
@@ -453,6 +453,7 @@ def main(args=None):
     commands = [common.NODE_BIN_PATH]
     if parsed_args.debug_mode:
         commands.append('--inspect-brk')
+    commands.append('--unhandled-rejections=strict')
```

Comment:
@U8NWXD, is this an expected change? if yes, can you please add a code comment above explaining why we are using this flag?

Reply:
done!